test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
no when user is created it will give some id that is Integer by some regex or a kind of validator, but not his id just an id that is fake